hbase-backup/src/main/java/org/apache/hadoop/hbase/backup/impl/BackupSystemTable.java

```diff
@@ -1885,6 +1887,13 @@ private String cellKeyToBackupSetName(Cell current) {
     return Bytes.toString(data).substring(SET_KEY_PREFIX.length());
   }
 
+  private void executeBufferedMutations(Table table, List<? extends Mutation> mutations)
```

Comment:
This helper function is now kinda weird. I think that you can replace all the above `try (Table table = ...) { ... }` with `try (BufferedMutator bm = ...) { ... }`. On quick glance, you never use the `table` instance other than to get the name in order to create the `BufferedMutator` instance.